src/providers/wms/qgswmscapabilities.h

```diff
@@ -933,6 +933,8 @@ class QgsWmsSettings
     QStringList mActiveSubLayers;
     QStringList mActiveSubStyles;
 
+    QStringList mOpacities;
```

Comment:
Can you add a docstring here (or add documented getters/setters, I am not completely familiar with the best practice with direct access to protected members in this area of the code)?